components/scream/cime_config/config_compsets.xml

```diff
@@ -33,4 +33,16 @@
       <support_level>Experimental, under development</support_level>
   </compset>
 
+  <compset>
+      <alias>F2010-SCREAMv1-noAero</alias> <!-- Coupled land-atm compset (using SCREAMv1), 2010 initial conditions, turn off Aerosol forcing -->
+      <lname>2010_SCREAM%noAero_ELM%SPBC_CICE%PRES_DOCN%DOM_SROF_SGLC_SWAV_SIAC_SESP</lname>
+      <support_level>Experimental, under development</support_level>
+  </compset>
+
+  <compset>
+      <alias>F2000-SCREAMv1-AQP1-noAero</alias> <!-- atm aquaplanet compset (using SCREAMv1), 2000 initial conditions, turn of Aerosol forcing -->
+      <lname>2000_SCREAM%no_Aero%AQUA_SLND_SICE_DOCN%AQP1_SROF_SGLC_SWAV</lname>
```

Comment:
In the F2010 compset, you call the physics option `noAero`, but in the AQP1 you call it `no_Aero`. I think these should be consistent (and probably one of them doesn't work with your regex in `namelist_deafults_scream.xml`.